src/main/java/it/pagopa/pn/deliverypush/service/impl/WebhookServiceImpl.java

```diff
@@ -173,12 +176,22 @@ private Mono<Void> processEvent(StreamEntity stream,  String oldStatus, String n
         }
 
         String timelineEventCategory = timelineElementInternal.getCategory().getValue();
+
+        List<String> filteredValues = new ArrayList<>();
+        if (eventType == StreamCreationRequest.EventTypeEnum.TIMELINE) {
+            filteredValues.addAll(stream.getFilterValues().isEmpty()
+                ? categoriesByVersion(TimelineElementCategoryInt.VERSION_10)
```

Comment:
ok come meccanismo, ma al posto di copiare addAll per ogni richiesta, io farei una lista costante inizializzata nel costruttore e poi in fase di check, va ad usarla solo se i filter son vuoti